webapp/src/index.tsx

```diff
@@ -7,10 +7,10 @@ import {
   Landing,
   Play,
   Users,
-  UserPage,
+  User,
```

Comment:
I see that it works, because we are not explicitly stating the types of the generics we use, and we rely on type inference for some returns (someone has broken something, but it's already in the main branch, so it's unrelated to your changes), but I'd still revert this particular change, otherwise it will stay fragile. This one, or the name of the User class in transcendence-app.

Reply:
I works but it could be confusing in some occasions. For me its's ok.
Up to you!

Reply:
We could, otherwise:

> O también podríamos en el back-end ponerles nombres mas específicos a los tipos de respuestas, por ejemplo UserHttpResponse, o algo parecido UserDtoResponse?

Reply:
I've been investigating this and it seems there is no conflicts due to having our components as functional and being the User model an interface. Typescript, merge declared types which fall in the same category but a function and an interface doesn't -> https://www.typescriptlang.org/docs/handbook/declaration-merging.html

That being said, understand why this was done, agree and will revert. But, if you don't mind, I will also append a "page" suffix to other page components, best to have a unified naming among similar components (in this case, pages)